### ISSUE TITLE
iOS: Failed validation for App Store Connect due to missing CFBundleVersion value

### ISSUE THREAD

**Issue by kolotum** [cofu-app/cbl-dart#81]
The validate app command failed for our app after we have integrated the cbl_flutter package. So we can not upload our iOS app update to App Store Connect:

> This bundle Payload/Runner.app/Frameworks/CouchbaseLite.framework is invalid. The Info.plist file is missing the required key: CFBundleVersion.

couchbase-lite-C is integrated as a Xcode sub project and its placeholders `$(CBL_VERSION_STRING)` and `$(CBL_BUILD_NUMBER)` in 
./native/vendor/couchbase-lite-C/Xcode/Framework-Info.plist are not properly replaced.

One workaround could be to replace these placeholders with dummy values, e.g. add command to build_apple.sh:

```bash
function fixVendorCouchbaseLiteCVersion() {
  cd "$nativeDir/vendor/couchbase-lite-C/Xcode"
  shortVersionStringPlaceholder='$(CBL_VERSION_STRING)'
  versionStringPlaceholder='$(CBL_BUILD_NUMBER)'
  sed -i '' "s/$shortVersionStringPlaceholder/1.0/" Framework-Info.plist
  sed -i '' "s/$versionStringPlaceholder/1/" Framework-Info.plist
}
```

And call this before running `./native/tools/build_apple.sh buildPlatform ${{ matrix.platform }}` in github workflow:
`./native/tools/build_apple.sh fixVendorCouchbaseLiteCVersion`

**Comment by blaugold** [comment]
Hi @kolotum, I'm sorry you're having this issue, and thanks for providing those details. I'll try to publish a fix today.

**Comment by blaugold** [comment]
A fix has been published in `cbl_flutter: ^0.4.0+3`.

**Comment by kaptnkoala** [comment]
Thanks for fixing this so quickly!!
